docs/rst/tutorials/core/qos/initial_peers/initial_peers.rst

```diff
@@ -0,0 +1,219 @@
+
+.. code-block:: bash
+
+      export FASTRTPS_DEFAULT_PROFILES_FILE=<path_to_xml_file>
+
+* **Alternative**: Placing the XML file in the running application directory under the name *DEFAULT_FASTRTPS_PROFILES.xml*.
+
+For example:
+
+.. code-block:: bash
+
+    cp <path_to_xml_file> DEFAULT_FASTRTPS_PROFILES.xml
+    export RMW_FASTRTPS_USE_QOS_FROM_XML=1
+
+.. important::
+
+    Repeat the same procedure in the other opened Vulcanexus docker instance.
+
+.. _initial_peers_example:
+
+Example
+-------
+
+Once both Vulcanexus docker environments have been configured, run a demo application in each instance:
+
+.. code-block:: bash
+
+      # Talker
+      ros2 run demo_nodes_cpp talker
+
+      # Listener
+      ros2 run demo_nodes_cpp listener
+
+The Vulcanexus docker instances would discover each other via multicast, then one of them would let the other know its address and port through it would establish any communication, and they will connect.
+
+.. _initial_peers_unicast_example:
+
+Unicast Example
+^^^^^^^^^^^^^^^
+
+If unicast is required instead of multicast, the following profile should be updated in the existing XML profile:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8" ?>
+        <profiles xmlns="http://www.eprosima.com/XMLSchemas/fastRTPS_Profiles">
+            <participant profile_name="initial_peers_example_profile" is_default_profile="true">
+                <rtps>
+                    <builtin>
+                        <initialPeersList>
+                            <locator>
+                                <udpv4>
+                                    <address>172.17.0.3</address> <!-- "keen_john"  IP address -->
+                                    <port>7777</port>             <!-- "keen_john"  port       -->
+                                </udpv4>
+                            </locator>
+                        </initialPeersList>
+                        <metatrafficUnicastLocatorList>
+                            <locator>
+                                <udpv4>
+                                    <address>172.17.0.2</address> <!-- "eager_pike" IP address -->
+                                    <port>7666</port>             <!-- "eager_pike" port       -->
+                                </udpv4>
+                            </locator>
+                        </metatrafficUnicastLocatorList>
+                    </builtin>
+                </rtps>
+            </participant>
+        </profiles>
+
+In this case, the port `7777` is not longer the default port, it is a new custom port to reach the peer address.
+Furthermore, the meta-traffic unicast locator has been set with the Vulcanexus docker instance IP address (own address), and the port it would be communicating through.
```

Comment:
Comment first what the ``metatrafficUnicastLocatorList`` does. Then explain that the same will apply to the other so to reach it out we should change the initial peer configuracion.